bmt/toolkit.py

```diff
@@ -343,34 +331,63 @@ def get_associations(
             A list of elements
 
         """
-        elements = self.get_descendants("association")
+        association_elements = self.get_descendants("association")
         filtered_elements: List[str] = list()
+        inverse_predicates: Optional[List[str]] = None
+        if predicates:
+            inverse_predicates = list()
+            for pred_curie in predicates:
+                predicate = self.get_element(pred_curie)
+                if predicate:
+                    inverse_p = self.get_inverse(predicate.name)
+                    if inverse_p:
+                        inverse_predicates.append(inverse_p)
+                inverse_predicates = self._format_all_elements(elements=inverse_predicates, formatted=True)
+
+        def match_association(
```

Comment:
can you pull this out into a standalone method instead of an internal method?

Reply:
Of course I could do that, if you think that is a useful extra method for 3rd party use.